alpha/apps/kaltura/lib/myEntryUtils.class.php

```diff
@@ -688,8 +688,74 @@ public static function createThumbnailFromEntry ( entry $entry , entry $source_e
 		}
 		return true;
 	}
-	
-	
+
+
+	public static function createThumbPaths($entity, $thumbNameAttributes, $thumbFilename, $format, $version, $thumbDirs, $contentPath, $captureRequest = false)
+	{
+		//create final path for thumbnail created
+		$finalThumbPath = myContentStorage::getThumbEntityPath(self::THUMB_ENTITY_NAME_PREFIX . $thumbDirs[0], $entity, $entity->getId() . $thumbNameAttributes, $thumbFilename, $version);
```

Comment:
there is a difference between the old flow and the new one
before the change the shared path that returned was for example:
/tempthumb/ . $dir . '/' . @thumbNameThatIncludesThumbAttributes@...
after the change we will have the path with partial file name. we should keep the old convention

Reply:
The difference between the paths is that in capture thumbnail we replace the entry Id with flavor asset Id
prev: 
entry Id 0_n0zkpsho: 

0_n0zkpsho_125_0_1__ffffff_0_0_0_0_0_-1_-1_-1_2_100002.jpg   => http://salma-ubuntu/p/102/thumbnail/entry_id/0_n0zkpsho/width/125
0_n0zkpsho_125_0_1__ffffff_0_50_50_0_0_-1_-1_-1_2_100002.jpg => http://salma-ubuntu/p/102/thumbnail/entry_id/0_n0zkpsho/width/125/src_x/50/src_y/50

0_n0zkpsho_125_0_1__ffffff_0_0_0_0_0_-1_-1_10_2_100002.jpg  => http://salma-ubuntu/p/102/thumbnail/entry_id/0_n0zkpsho/width/125/vid_slices/10
0_n0zkpsho_125_0_1__ffffff_0_0_0_0_0_-1_4_10_2_100002.jpg   => http://salma-ubuntu/p/102/thumbnail/entry_id/0_n0zkpsho/width/125/vid_slice/4/vid_slices/10
0_n0zkpsho_125_0_1__ffffff_0_0_0_0_0_-1_14_10_2_100002.jpg  => http://salma-ubuntu/p/102/thumbnail/entry_id/0_n0zkpsho/width/125/vid_slice/14/vid_slices/10 
0_n0zkpsho_125_0_1__ffffff_0_0_0_0_0_5_-1_-1_2_100002.jpg   => http://salma-ubuntu/p/102/thumbnail/entry_id/0_n0zkpsho/width/125/vid_sec/5

after change: 
entry Id 0_ksxyggf2, flavor Id 0_3t5rng0i: 

0_ksxyggf2_125_0_1__ffffff_0_0_0_0_0_-1_-1_-1_2_100002.jpg    => http://salma-ubuntu/p/102/thumbnail/entry_id/0_ksxyggf2/width/125
0_ksxyggf2_125_0_1__ffffff_0_50_50_0_0_-1_-1_-1_2_100002.jpg  => http://salma-ubuntu/p/102/thumbnail/entry_id/0_ksxyggf2/width/125/src_x/50/src_y/50

0_3t5rng0i_125_0_1__ffffff_0_0_0_0_0_-1_-1_10_2_3.jpg   => http://salma-ubuntu/p/102/thumbnail/entry_id/0_ksxyggf2/width/125/vid_slices/10
0_3t5rng0i_125_0_1__ffffff_0_0_0_0_0_-1_4_10_2_3.jpg    => http://salma-ubuntu/p/102/thumbnail/entry_id/0_ksxyggf2/width/125/vid_slice/4/vid_slices/10
0_3t5rng0i_125_0_1__ffffff_0_0_0_0_0_-1_10_10_2_3.jpg   => http://salma-ubuntu/p/102/thumbnail/entry_id/0_ksxyggf2/width/125/vid_slice/14/vid_slices/10
0_3t5rng0i_125_0_1__ffffff_0_0_0_0_0_5_-1_-1_2_3.jpg    =>  http://salma-ubuntu/p/102/thumbnail/entry_id/0_ksxyggf2/width/125/vid_sec/5